DRAFT: Multi-Party Payout.md

```diff
@@ -0,0 +1,204 @@
+| scriptPayoutSize | compactSize uint | 1-9 | Size of the Payee Script. | DIP3 |
+| scriptPayout | Script | Variable | Payee script (p2pkh/p2sh) | DIP3 |
+| inputsHash | uint256 | 32 | The SHA256 hash of all the outpoints of the transaction inputs | DIP3 |
+| payloadSigSize | compactSize uint | 1-9 | Size of the Signature | DIP3 |
+| payloadSig | unsigned char[] | Variable | Signature of the hash of the ProTx fields. Signed by the Owner. | DIP3 |
+
+
+## Validation Rules
+
+### ProRegTx and ProUpRegTx
+
+A ProRegTx is invalid if any of these conditions are true (all defined in DIP3 except conditions 11-13):
+
+  1. collateralOutpoint `hash` is null but an output with 1000 DASH is not present at position `n` of the ProRegTx outputs
+  2. collateralOutpoint `hash` is not null but an output with 1000 DASH can't be found in the UTXO specified by the `hash` and `n`
+  3. Any KeyId* field is null (KeyIdOwner, KeyIdOperator or KeyIdVoting)
+  4. KeyIdOwner or PubKeyOperator was already used by any entry in the registered masternodes set
+  5. scriptPayout is not a P2PKH or P2SH script
+  6. When scriptPayout is P2PKH script and the public key hash equals any of KeyIdOwner or KeyIdVoting
+  7. ipAddress is set and port is not set to the default mainnet port
+  8. ipAddress is set and not routable or not an IPv4 mapped address
+  9. ipAddress is set and already used in the registered masternodes set
+  10. operatorReward > 10000
+  11. sharedReward > 10000 (Current DIP)
+  12. sharedReward > 0 and shareHolders field is null (Current DIP)
+  13. sum of values in shareHolders field != 100 (Current DIP)
+  14. The inputsHash does not match the calculated hash
+  15. collateralOutpoint `hash` is null and payloadSig is not empty (zero size)
+  16. collateralOutpoint `hash` is not null and payloadSig is not a valid signature signed with the collateral key
+  17. collateralOutpoint `hash` is not null and the referenced collateral is not a P2PKH output
+
+Please note that while deploying this DIP, additional and temporary validation rules may apply. The details of these temporary rules will be described in the deployment plan.
+
+### ProUpRevTx
+
+A ProUpRevTx is invalid if any of these conditions are true (all defined in DIP3 except conditions 2-4):
+
+1. proTxHash can not be found in the registered masternode set
+2. sharedReward > 10000 (Current DIP)
+3. sharedReward > 0 and shareHolders field is null (Current DIP)
+4. sum of values in shareHolders field != 100 (Current DIP)
+5. The inputsHash does not match the calculated hash
+6. payloadSig is invalid
+
+## Masternode Rewards
+
+The new system allows to deterministically ascertain the recipient of the masternode’s portion of the next block reward (DIP3) beyond the owner/operator paradigm, but doesn't change any of the mechanisms to determine reward payment.
+
+The rules to determine the next block’s payee are will continue to be (DIP3):

```

Comment:
I would prefer to simply link to the relevant DIP3 section here rather duplicating it. In the (unlikely) event DIP3 is ever changed we could end up with a discrepancy between the 2 documents.
```suggestion
The rules to determine the next block’s payee will continue to be the same as [originally defined in DIP3](https://github.com/dashpay/dips/blob/master/dip-0003.md#masternode-rewards).
```